app/controllers/pages_controller.rb

```diff
@@ -58,7 +58,8 @@ def patrons
              .select("CEIL(100 * LN(COUNT(referees.id) + 1)) AS aura")
              .joins("LEFT JOIN users referees ON users.id = referees.referrer_id")
              .group("users.id")
-             .order(aura: :desc)
+             .having("CEIL(100 * LN(COUNT(referees.id) + 1)) > 0")
+             .order(aura: :desc, referrals: :desc)
```

Comment:
If people have the same aura then they'll have the same number of referrals too.
If you want to add deep ordering I would do aura DSC then username ASC instead